README.rst

```diff
@@ -162,6 +162,67 @@ To understand the details of this statement, take a look at the code of
 `django_loci.storage.OverwriteStorage
 <https://github.com/openwisp/django-loci/blob/master/django_loci/storage.py>`_.
 
+``DJANGO_LOCI_GEOCODER``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``str``     |
++--------------+-------------+
+| **default**: | ``ArcGIS``  |
++--------------+-------------+
+
+Service used for geocoding and reverse geocoding.
+
+Supported geolocation services:
+
+* `ArcGIS`
+* `Nominatim`
+* `GoogleV3` (Google Maps v3)
+
+``DJANGO_LOCI_GEOCODE_STRICT_TEST``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    | ``bool``  |
++--------------+-----------+
+| **default**: | ``True``  |
++--------------+-----------+
+
+If geocoding tests failed and set to ``False`` won't raise error.
+
+``DJANGO_LOCI_GEOCODE_FAILURE_DELAY``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+----------+
+| **type**:    | ``int``  |
++--------------+----------+
+| **default**: | ``1``    |
++--------------+----------+
+
+Amount of seconds between geocoding retry API calls if geocoding request failed.
+
+``DJANGO_LOCI_GEOCODE_RETRIES``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+----------+
+| **type**:    | ``int``  |
++--------------+----------+
+| **default**: | ``3``    |
++--------------+----------+
+
+Amount of retry API calls if geocoding request failed.
+
+``DJANGO_LOCI_GEOCODE_GOOGLEV3_API_KEY``
```

Comment:
rename this to `DJANGO_LOCI_GEOCODE_API_KEY`, we may add support for other services which also require an API key so this should be generic